baselines/dom.generated.d.ts

```diff
@@ -17647,6 +17650,7 @@ type CanvasTextBaseline = "top" | "hanging" | "middle" | "alphabetic" | "ideogra
 type ChannelCountMode = "max" | "clamped-max" | "explicit";
 type ChannelInterpretation = "speakers" | "discrete";
 type ClientTypes = "window" | "worker" | "sharedworker" | "all";
+type ColorSpaceConversion = "none" | "default";
```

Comment:
Small global aliases are annoying because of alias interning: everybody with a type none | default will now see the name ColorSpaceConversion for it. 

Not sure the best workaround for this. I’ll think about it.